frontend/locales/en-US.json

```diff
@@ -442,6 +443,8 @@
   "signIn": "Sign in",
   "snackbar": {
     "addedToQueue": "Added to queue",
+    "instantMixFailed": "Failed to add Instant mix to queue",
```

Comment:
Same thing I mentioned in #1964 about the error keys

Reply:
@noaione Sorry for forgetting about telling this before: Put all the new keys in all your PRs at the root level please. I'm trying to remove a lot of duplication around keys and I found out that most of that duplication is done blindly while using subkeys.

I have a branch ready locally to remove all the nesting after all your PRs are in :)